### ISSUE TITLE
cant integrate bitwarden, AD & Bitwarden directory connector 

### ISSUE THREAD

**Issue by sarojamarraj** [bitwarden/directory-connector#5]
bitwarden is installed on On-premises Ubuntu 16.04 ( Docker )

Can access Bitwarden on web interface, Test Organization is also created 

now i want to integrate it with AD, i have installed bitwarden connector on my laptop ( windows 10 )
but i can't login to bitwarden account via connector.

i can access bitwarden on premises server via FQDN 

What Port is used by bitwarden directory connector ?

Any guild-lines or steps on how to integrate Bitwarden connector with bitwarden on premises?

**Comment by kspearrin** [comment]
Did you follow the documentation and configure the environment? https://help.bitwarden.com/article/directory-sync/#configure-environment

**Comment by sarojamarraj** [comment]
Yes i followed the documents correctly, there was issues with access for my username.
i have moved a bit ahead, now When i sync it says username or password are not configured! even though all the required information are configured.
Which file does the connector makes the changes to?
Where does the connector saves the config?
